### PR TITLE
music-release-project

### PR DESCRIPTION
**Pull request by JoacimNilsen**
Link to my deployed project:
https://peaceful-lichterman-5a8ff7.netlify.com/

My reflections on how this week's project turned out:
I am quite satisfied with how it turned out, since i was so freaked out and found React so hard and abstract to comprehend. But after Damiens codealong it made some more sense. I wish I had time to do the stretch goal this week because I really enjoyed working on this project. 

Things I'd like to have clarified or explained in more detail:

### CODE REVIEW

File: code/src/App.js [hunk 1]
```diff
@@ -1,12 +1,19 @@
...
+import { Album } from './components/Album'
 
 export const App = () => {
-  return (
-    <div>
-      Find me in src/app.js!
-    </div>
-  )
+	return (
+		<React.Fragment>
```
Comment (JennieDalgren) [inline code/src/App.js]: nice 😎

File: code/src/app.css [hunk 2]
```diff
@@ -0,0 +1,17 @@
+.contentWrapper {
+	display: flex;
+	flex-wrap: wrap;
+	justify-content: center;
+}
+
+header {
+	margin: 0 35px 0 35px;
```
Comment (JennieDalgren) [inline code/src/app.css]: shortcut for this is: `margin: 0 35px;`

File: code/src/components/Album.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+			<Logos />
+		</div>
+		<div className='titleWrapper'>
+			<a className='albumName' href={props.item.external_urls.spotify}>
+				{props.item.name}
+			</a>
+			<div className='artistName'>
+				{props.item.artists.map(item => {
+					return <Artist key={item.id} item={item} />
```
Comment (JennieDalgren) [inline code/src/components/Album.js]: here you actually just need to pass along the artist name and the url. so no need to pass the whole item object. But it seems like it only shows one artist, even though there are multiple artists in the array..?

**Comment by JoacimNilsen** [comment]
Thank you!
Fixed the indentation, I had it on 2 in prettier but i also had to do it in
vs code it seemed :)

I will look in to that! This week is swamped because my son is going to
have his tonsils removed so my focus is elsewhere right now. My mentor is
coming to visit me this week and I will have a chat with him about it.
Cheers!

Den fre 29 nov. 2019 kl 11:32 skrev Jennie Dalgren <notifications@github.com

**Pull request closed without merging**